ATCBot/SteamConfig.cs

```diff
@@ -0,0 +1,49 @@
+using System.IO;
+using Newtonsoft.Json;
+
+namespace ATCBot
+{
+    public class SteamConfig
+    {
+        public static SteamConfig Config;
+        public string SteamUserName;
+        public string SteamPassword;
+        public string TwoFactorAuthCode;
+        public string AuthCode;
```

Comment:
While not required (I can fill it in myself), can you add XML docs to these members?